README.md

```diff
@@ -3,3 +3,15 @@
 The _Official_ Website of the Nevada County Free Code Camp.
 
 https://nevadacountyfreecodecamp.github.io/
+
+
+# Changes since May 30th
+Included normalize.css for style reset
+
+Went with Expletus and Raleway fonts as an idea, could be anything else though
+
+Wouldn't mind switching to SCSS and maybe something like bourbon/neat etc to practice something like that if other people are into it. Or not, and stick to basic CSS
```

Comment:
@andreas-groos you can just make a new commit on your `new-layout` branch, then push, which will update this Pull Request. PRs are updated with new commits as long as they are still open / unmerged.